docs/data/joy/components/css-baseline/css-baseline.md

```diff
@@ -0,0 +1,168 @@
+---
+product: joy-ui
+githubLabel: 'component: CssBaseline'
+---
+
+# CSS Baseline
+
+<p class="description">Joy UI provides a CssBaseline component to kickstart an elegant, consistent, and simple baseline to build upon.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Global reset
+
+You might be familiar with [normalize.css](https://github.com/necolas/normalize.css), a collection of HTML element and attribute style-normalizations.
```

Comment:
We probably should specify that it is used, this is why we are mentioning it :)

Reply:
I think it is mentioned to give the reader a clue that it is a similar purpose. I checked `normalize.css` and we are not using the same global reset.